server/sonar-server/src/main/java/org/sonar/server/startup/RegisterDashboards.java

```diff
@@ -55,7 +51,7 @@
   private final LoadedTemplateDao loadedTemplateDao;
 
   public RegisterDashboards(DashboardTemplate[] dashboardTemplatesArray, DashboardDao dashboardDao,
-    ActiveDashboardDao activeDashboardDao, LoadedTemplateDao loadedTemplateDao) {
+    ActiveDashboardDao activeDashboardDao, LoadedTemplateDao loadedTemplateDao, RegisterIssueFilters startupDependency) {
```

Comment:
Please add a comment to explain why RegisterIssueFilters is needed only in the constructor